File: QhTemplate.ApplicationService/PyScripts/BriefingContent.py
class BriefingContent():
    # 公司名称
    CompanyName=""
    # 举办学校
    School=""
    # 举办教学楼
    Held=""
    # 宣讲时间
    StartTime=""
    # 发布时间
    PublishTime=""
    # 宣讲正文
    Body=""
    # 操作链接
    OpthonList=""
    # 举办城市
    City=""

    def __init__(self, company,city,school,held,starttime,publishtime,body,optionlist):
        self.CompanyName=company
        self.City=city
        self.School=school
        self.Held=held
        self.StartTime=starttime
        self.PublishTime=publishtime
        self.Body=body
        self.OpthonList=optionlist

File: QhTemplate.MysqlEntityFrameWorkCore/Models/BriefingContent.cs
﻿using System;
using System.Collections.Generic;
using System.Runtime.CompilerServices;

namespace QhTemplate.MysqlEntityFrameWorkCore.Models
{
    public partial class BriefingContent
    {
        public int Id { get; set; }
        public string CompanyName { get; set; }
        public int? SchoolId { get; set; }
        public string Held { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public string OpthonList { get; set; }
        public DateTime PublishTime { get; set; }
        public DateTime StartTime { get; set; }

        public IEnumerable<NoticeBriefing> NoticeBriefings { get; internal set; }

        public static BriefingContent Create(string title, string content, string held,
            DateTime startTime, int schoolId,string companyName)
        {
            return new BriefingContent
            {
                Held = held,
                SchoolId = schoolId,
                Title = title,
                Content = content,
                StartTime = startTime,
                PublishTime = DateTime.Now,
                CompanyName=companyName
            };
        }

        public void Update(string title, string content, string held,
            DateTime startTime)
        {
            this.Content = content;
            this.Title = title;
            this.StartTime = startTime;
            this.Held = held;
        }
    }
}
